### PR TITLE
feat: How to Delete an account

### PR DESCRIPTION
**Pull request by andersc2022**
This guide show how to Delete an account in CCMP

build: Bump version tags for GitHub actions

GitHub advises against the use of actions using Node.js 12. This is true for the v2 releases of the "checkout" and "setup-python" actions. Thus, bump those actions to their latest versions (v3 and v4, respectively).

While we're at it, bump the "upload-artifact" action to v3 as well.

Reference:
https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/

### CODE REVIEW

**Comment by fghaas** [comment]
I see that this PR is open, but it's not based on current `main`, it doesn't have a request for review, and also it doesn't contain any new text except a heading with no further content.

@andersc2022 Did you mean to open this as a Draft PR, perhaps?

**Comment by andersc2022** [comment]
@fghaas I had a little problem but I think have solved it now.

**Comment by andersc2022** [comment]
I hope and think my guide is ready for a Review :-)

File: docs/howto/account-billing/Delete-account.md
Comment (fghaas) [inline docs/howto/account-billing/Delete-account.md:19]: These sound awkward and will need rephrasing. I'll leave it to @colder-is-better to provide suggestions. :)

**Review by fghaas — COMMENTED** [review]
I've added some thoughts; I guess @colder-is-better will have some more. :)

File: docs/howto/account-billing/Delete-account.md
Comment (colder-is-better) [inline docs/howto/account-billing/Delete-account.md:17]: Assuming all user assets will be wiped in 72 hours, I would write this as follows:

```suggestion
Any assets you may still have in {{brand}} will be wiped after 72 hours. Please keep in mind that during that time you will be charged as usual.
```

**Review by colder-is-better — COMMENTED** [review]
Looking good (and for now, I only have two suggestions :))

File: docs/howto/account-billing/Delete-account.md
Comment (andersc2022) [reply docs/howto/account-billing/Delete-account.md:17]: No, the assets will only be suspended after 3 days. After 10 days we will start deleting servers and other stuff.

Shall I instead write 

Any assets you may still have in {{brand}} will be wiped after 10 days. Please keep in mind that during that time you will be charged as usual.

File: docs/howto/account-billing/Delete-account.md
Comment (andersc2022) [reply docs/howto/account-billing/Delete-account.md:19]: Sound better - i fix

Comment (andersc2022) [reply docs/howto/account-billing/Delete-account.md:19]: You want me to add red color on the words **If** and **and** ?

Comment (andersc2022) [reply docs/howto/account-billing/Delete-account.md:19]: It´s me who wanna save any customer at any price :-)

Comment (andersc2022) [reply docs/howto/account-billing/Delete-account.md:19]: But colder-is-better wrote it better

Comment (andersc2022) [reply docs/howto/account-billing/Delete-account.md:19]: If wee choose 10 days should maybe the last text be:

If you change your mind during the first 3 days, contact us by emailing support@cleura.com and we will **not delete** your account.

File: docs/howto/account-billing/Delete-account.md
Comment (colder-is-better) [reply docs/howto/account-billing/Delete-account.md:17]: Aha, so assets get wiped irrevocably after 10 and not after 3 days. Does this make any difference in the pricing structure? Is the customer getting charged as usual during this 10-day time period? If so, then I would suggest this change:

 ```suggestion
Any assets you may still have in {{brand}} will be suspended after 3 days, and wiped irrevocably 7 days after suspension. Please be aware that during these 10 days, you will be getting charged as usual.
```

File: docs/howto/account-billing/Delete-account.md
Comment (colder-is-better) [reply docs/howto/account-billing/Delete-account.md:19]: OK, so even though assets are irrevocably wiped after 10 days, the customer has only 3 days to change their mind. If this is the case indeed, then taking into account Florian's suggestion for contacting support, I would suggest this change:

```suggestion
If you change your mind during the first 3 days please [contact support](https://servicecenter.citynetwork.eu/servicedesk), and we will **not delete** any of your assets, nor your account.```

File: docs/howto/account-billing/Delete-account.md
Comment (andersc2022) [reply docs/howto/account-billing/Delete-account.md:13]: Of course, my fault :-)

File: docs/howto/account-billing/Delete-account.md
Comment (colder-is-better) [inline docs/howto/account-billing/Delete-account.md:3]:
```suggestion
In case you decide to close your account in {{brand}}, you may do so via the [{{gui}}](https://{{gui_domain}}).
```

File: docs/howto/account-billing/Delete-account.md
Comment (colder-is-better) [inline docs/howto/account-billing/Delete-account.md:8]:
```suggestion
A pane named _Account settings_ will slide over from the right-hand side of the browser window. To delete your account, click the _Settings_ tab.
```

File: docs/howto/account-billing/Delete-account.md
Comment (colder-is-better) [inline docs/howto/account-billing/Delete-account.md:12]:
```suggestion
Scroll to the bottom of the pane, and click the red button named _Delete Account_.
```

File: docs/howto/account-billing/Delete-account.md
Comment (colder-is-better) [inline docs/howto/account-billing/Delete-account.md:16]:
```suggestion
Choose a reason for leaving and type in your current password. Click the red button named _Delete Account_. That will create a request for account deletion.
```

**Review by colder-is-better — COMMENTED** [review]
We are on a good track, and I only have a couple of suggestions.

File: docs/howto/account-billing/Delete-account.md
Comment (colder-is-better) [inline docs/howto/account-billing/Delete-account.md:16]: Since this is the second time we click on the red button, I suggest the following _slight_ change:

```suggestion
Choose a reason for leaving and type in your current password. Click the red button named _Delete Account_ one more time. That will create a request for account deletion.
```

File: docs/howto/account-billing/Delete-account.md
Comment (colder-is-better) [inline docs/howto/account-billing/Delete-account.md:24]: To be consistent with how we specify the time (in days), I'm now suggesting this:

```suggestion
If you change your mind during the first three (3) days, please get in touch with [Service Center](https://servicecenter.citynetwork.eu/servicedesk), and we will **not delete** your assets or your account.
```

I also went with "get in touch", for I find it friendlier than "contact" :)

**Comment by colder-is-better** [comment]
Looks like we're good to go --- thank you, @andersc2022! Before we merge, though, let us have @fghaas's point of view.

File: docs/howto/account-billing/Delete-account.md
Comment (fghaas) [inline docs/howto/account-billing/Delete-account.md:24]:
```suggestion
If you change your mind during the first three (3) days, please get in touch with our [{{support}}](https://{{support_domain}}/servicedesk), and we will **not delete** your assets or your account.
```

File: docs/howto/account-billing/Delete-account.md
Comment (fghaas) [inline docs/howto/account-billing/Delete-account.md:3]:
```suggestion
In case you decide to close your {{brand}} account, you may do so via the [{{gui}}](https://{{gui_domain}}).
```

**Review by fghaas — COMMENTED** [review]
Couple more small issues. Please fix, and then we can merge this.

**Comment by andersc2022** [comment]
Hi Florian

I don´t believe it :-)

Did your two changes and then upload it but got this error "failed to push some refs ?

Have you seen this error before ?


Picture

Can it be something with the link ?

I change the link from

[Service Center](https://servicecenter.citynetwork.eu/servicedesk)

to

[{{support}}](https://{{support_domain}}/servicedesk)

I´m lost :-)

**Comment by fghaas** [comment]
@andersc2022 You can't use Git with GitHub at this time; it's just Hub. https://www.githubstatus.com/

**Comment by fghaas** [comment]
@andersc2022 You can't use Git with GitHub at this time; it's just Hub. https://www.githubstatus.com/

**Comment by fghaas** [comment]
@andersc2022 You can retry pushing now; GitHub appears to be feeling better.

**Comment by andersc2022** [comment]
Yeees, It went very good now again. All commands was succesful.

**Review by fghaas — APPROVED** [review]
This looks good to me now, thanks!

**Pull request merged**
